ui/components/multichain/multichain-accounts/wallet-details-account-item/wallet-details-account-item.tsx

```diff
@@ -61,14 +57,9 @@ const WalletDetailsAccountItem = ({
       {...rowStylesProps}
     >
       <Box display={Display.Flex} alignItems={AlignItems.center} gap={3}>
-        <AvatarAccount
+        <PreferredAvatar
```

Comment:
Can we ensure that Jazzicons are rendered correctly by the DS component before using it everywhere? IIRC there were some issues with not consistently rendering non-EVM addresses

Reply:
Addressed by #35748